ccd-definition/CaseEventToFields/CareSupervision/manageOrders/4_childrenDetailsSection.json

```diff
@@ -48,7 +48,7 @@
     "PageDisplayOrder": 5,
     "PageColumnNumber": 1,
     "FieldShowCondition": "orderAppliesToAllChildren=\"No\"",
-    "ShowSummaryChangeOption": "Y"
+    "ShowSummaryChangeOption": "N"
```

Comment:
bug that Rob spotted, list of all children is shown in CYA but this is confusing because you only want to see the ones you selected (although you don't see that either because of XUI bug)

Reply:
Great highlight